waiter/src/waiter/state.clj

```diff
@@ -270,10 +270,11 @@
    work-stealing-received-in-flight-counter requests-outstanding-counter {:keys [cid instance response-chan router-id] :as data}]
   (cid/cdebug cid "received work-stealing instance" (:id instance) "from" router-id)
   (counters/inc! (metrics/service-counter service-id "work-stealing" "received-from" router-id "offers"))
-  (if (pos?
-        (utils/compute-help-required
-          (counters/value slots-in-use-counter) (counters/value slots-available-counter)
-          (counters/value work-stealing-received-in-flight-counter) (counters/value requests-outstanding-counter)))
+  (if (utils/requires-help?
```

Comment:
Can we fix the docstring of `handle-work-stealing-offer`?  It never returns `nil`.